tools/istio-clean-iptables/pkg/cmd/cleanup.go

```diff
@@ -26,13 +29,19 @@ func flushAndDeleteChains(ext dep.Dependencies, cmd string, table string, chains
 	}
 }
 
-func removeOldChains(ext dep.Dependencies, cmd string) {
+func removeOldChains(cfg *config.Config, ext dep.Dependencies, cmd string) {
+	// Remove the old TCP rules
 	for _, table := range []string{constants.NAT, constants.MANGLE} {
-		// Remove the old chains
 		ext.RunQuietlyAndIgnore(cmd, "-t", table, "-D", constants.PREROUTING, "-p", constants.TCP, "-j", constants.ISTIOINBOUND)
 	}
 	ext.RunQuietlyAndIgnore(cmd, "-t", constants.NAT, "-D", constants.OUTPUT, "-p", constants.TCP, "-j", constants.ISTIOOUTPUT)
 
+	redirectDNS := cfg.RedirectDNS
+	// Remove the old DNS UDP rules
+	if redirectDNS {
+		common.HandleDnsUdp(common.DeleteOps, builder.NewIptablesBuilder(), ext, cmd, cfg.ProxyUID, cfg.ProxyGID, cfg.DNSServersV4)
```

Comment:
This ends up running `iptables iptables -t nat -D OUTPUT -p udp --dport 53 -m owner --uid-owner 1337 -j RETURN` (note 2x iptables)

Reply:
Good catch, thanks a lot! Making careless mistakes while wrapping up the code... :(